src/SFA.DAS.EmployerIncentives.UnitTests/Application/Services/CommitmentServiceTests/WhenApiIsUnavailable.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.EmployerIncentives.Application.Services;
+
+namespace SFA.DAS.EmployerIncentives.UnitTests.Application.Services.CommitmentServiceTests
+{
+    [TestFixture] 
+    public class WhenApiIsUnavailable
+    {
+        [Test]
+        public async Task Then_health_check_should_return_false()
```

Comment:
The title for this should be Pascal case, but also - and also the additional test coverage for when it is healthy is missing

Reply:
changed casing, this is covered in the Acceptance Tests